internal/aws/containerinsightcommon/const.go

```diff
@@ -107,11 +116,15 @@ var metricToUnitMap = make(map[string]string)
 
 func init() {
 	//cpu metrics
-	metricToUnitMap[CpuTotal] = UnitNanoSecond
-	metricToUnitMap[CpuUser] = UnitNanoSecond
-	metricToUnitMap[CpuSystem] = UnitNanoSecond
-	metricToUnitMap[CpuLimit] = UnitCount
+	//The following metrics are reported in unit of millicores, but cloudwatch doesn't support it
+	//  CpuTotal
+	//	CpuUser
+	//	CpuSystem
+	//	CpuLimit
+	//  CpuRequest
 	metricToUnitMap[CpuUtilization] = UnitPercent
+	metricToUnitMap[CpuReservedCapacity] = UnitPercent
```

Comment:
You can merge all the map assignment into one.

```
var metricsToUniMap ....

func init() {
   t := map[xxx]xxx{
        a: b,
        c: d,
   }
   metricsToUnitMap = t
}
```

Reply:
Good suggestion.